consume-lsp

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash
+
+# This script is meant to be run after the Flux release script has
+# opened a PR in the LSP, and that PR has merged.
+
+# It should handle the rest of the release process, with minimal 
+# babysitting by an engineer.
+
+# If any command fails, exit the script
+set -e
+
+# Check if hub is installed
+if ! command -v hub &> /dev/null
+then
+    echo "hub is not installed. exiting"
+fi
+
+# Make a temporary directory where we can clone fresh copies of all the necessary repos
+TEMPDIR=$(mktemp -d -t lsp-release.XXXX)
+echo "Using ${TEMPDIR}"
+cd $TEMPDIR
+
+# Cut down on code duplication
+function tmp_clone() {
+	git clone git@github.com:influxdata/$1.git &> /dev/null
+	echo "$(pwd)/$1"
```

Comment:
We could, but this is another case where I was using `echo` to mean "this function returns the following string." Figured it was easier to do this rather than the following:

```
LSP_DIR=$TEMP_DIR/flux-lsp
CLI_DIR=$TEMP_DIR/flux-lsp-cli
// etc...
``` 

in addition to all the `git clone` calls